lib/posticle/src/grammar.pest

```diff
@@ -103,9 +118,7 @@ terminator = _{
 }
 
 punctuation = _{
-    "-" | "_" | "!" | "\"" | "#" | "$" | "%" | "&" | "'" | "(" | ")" | "*" | "+" | "," | "."
-    | "/" | ":" | ";" | "<" | "=" | ">" | "?" | "@" | "[" | "]" | "^" | "`" | "{" | "|" | "}"
-    | "~"
+    terminator | "-" | "_" | "#" | "+" | "/" | "^" | "`" | "{" | "}" | "‹" | "›" | "»" | "«" | "「" | "」«" | "《" | "》" | "‘" | "’" | "‚" | "‛" | "“" | "”" | "„" | "‟" | "…"
```

Comment:
`... | "」«" | ...` looks like a typo; also, where did `"` and `'` go?

Reply:
Wow, yeah, that's a bad one. `"` and `'` are covered by terminator.

Reply:
wow, totally missed `terminator`, lmao. i should not be reviewing code and simultaneously trying to take notes during statistics discussion, haha

Reply:
And I shouldn't commit and push right before going to sleep...